ansible_builder/cli.py

```diff
@@ -8,6 +8,24 @@
 
 
 def prepare(args=sys.argv[1:]):
+    args = parse_args(args)
+    return AnsibleBuilder(**vars(args))
+
+
+def run():
+    args = parse_args()
+    if args.action in ['build', 'create']:
```

Comment:
looks like something else needs to be hooked up, trying to run this I get

```
$ ansible-builder introspect
An error has occured.
```

Reply:
Oops. This should be fixed now 😅